lib/mix/test/mix/tasks/format_test.exs

```diff
@@ -281,6 +281,45 @@ defmodule Mix.Tasks.FormatTest do
     end)
   end
 
+  defmodule Elixir.ExtensionSfacePlugin do
```

Comment:
I don't think we need this new module+case. Instead, you can check in the previous plugins that they are receiving the proper `:sigils` option in their `format/2`. WDYT?

Reply:
@josevalim Sure, on it :)